internal/commands/sastrealtime/sast_test.go

```diff
@@ -0,0 +1,22 @@
+package scarealtime
+
+import (
+	"os"
+	"testing"
+
+	"github.com/checkmarx/ast-cli/internal/commands/sastrealtime/sastconfiguration"
+	"github.com/checkmarx/ast-cli/internal/commands/scarealtime/scaconfiguration"
+	"github.com/checkmarx/ast-cli/internal/services/osinstaller"
+	"gotest.tools/assert"
+)
+
```

Comment:
For tests, please follow the convention:
UnitOfWork_StateUnderTest_ExpectedBehavior

UnitOfWork: This is the method or function being tested.
StateUnderTest: This describes the conditions under which the test is run.
ExpectedBehavior: This describes the expected outcome of the test.

Look at some references in project_test.go, first tests in result_test.go and some other files